commit 6f88e8f35adc9b6a1cdb3a6f8fd8d838292e7731
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Jan 28 14:42:34 2019 +0000

    input: rpi-ft5406: Clear build warning on 64 bit builds.
    
    Resolve 64 bit build warning over using %x with a dma_addr_t.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/input/touchscreen/rpi-ft5406.c b/drivers/input/touchscreen/rpi-ft5406.c
index 40bbde9ce1bc..4115b4666211 100644
--- a/drivers/input/touchscreen/rpi-ft5406.c
+++ b/drivers/input/touchscreen/rpi-ft5406.c
@@ -218,8 +218,8 @@ static int ft5406_probe(struct platform_device *pdev)
 
 	if (!ts->ts_base) {
 		dev_warn(dev,
-		"set failed, trying get (err:%d touchbuf:%x virt:%p bus:%x)\n",
-		err, touchbuf, ts->ts_base, ts->bus_addr);
+		"set failed, trying get (err:%d touchbuf:%x virt:%p bus:%pad)\n",
+		err, touchbuf, ts->ts_base, &ts->bus_addr);
 
 		err = rpi_firmware_property(
 				fw,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1326_6f88e8f35adc9b6a1cdb3a6f8fd8d838292e7731.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1326_6f88e8f35adc9b6a1cdb3a6f8fd8d838292e7731.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
